test/evergreen.yml

```diff
@@ -1319,12 +1319,12 @@ tasks:
         vars:
           CMAKE_TOOLCHAIN_FILE: -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/gcc.cmake
           GNU_C_VERSION: -DGNU_C_VERSION=8
-          DGNU_CXX_VERSION: -DDGNU_CXX_VERSION=8
+          GNU_CXX_VERSION: -DGNU_CXX_VERSION=8
```

Comment:
This typo caused some test failures after we started using the v4 toolchain. The extra D meant we never set `CXX_VERSION` in `configure wiredtiger` and fell back on the platform version

Reply:
Great catch. I also saw from WT-10642 that we are missing CLANG_C_VERSION and CLANG_CXX_VERSION from the  "configure wiredtiger" function. Might fix that while you are doing this?

Reply:
I gave it a try but it's producing some additional compilation errors that will need time to resolve.
I'll spin off a ticket for the CLANG vars so we can merge this PR and resolve the CI-blocker

Reply:
[WT-11239](https://jira.mongodb.org/browse/WT-11239) created